standard_renderer.go

```diff
@@ -268,6 +237,12 @@ func (r *standardRenderer) flush() {
 			}
 		}
 	}
+
+	// Clearing left over content from last render.
+	if r.linesRendered > numLinesThisFlush {
+		buf.WriteString(ansi.EraseDisplayRight)
```

Comment:
Could you rebase onto `main`?

```suggestion
		buf.WriteString(ansi.EraseDisplayBelow)
```